src/pages/ProfileSettings/utils/handleOptionsRender.tsx

```diff
@@ -0,0 +1,9 @@
+export const handleOptionsRender = (arr: any): [] => {
+    return arr.map((element: any, index: any) => {
+        return (
+            <option key={index} value={element.sigla}>
```

Comment:
Evitar usar o index como key. Segue explicação:

https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66